pkgs/development/libraries/glew/default.nix

```diff
@@ -1,6 +1,14 @@
-{ lib, stdenv, fetchurl, fetchpatch, cmake, libGLU, libXmu, libXi, libXext
+{ lib
+, stdenv
+, fetchurl
+, fetchpatch
+, cmake
+, libGLU
+, libXmu
+, libXi
+, libXext
 , OpenGL
-, enableEGL ? false
+, enableEGL ? true
```

Comment:
!hostPlatform.isDarwin, judging by `meta.platforms` below

Reply:
Made it `stdenv.isDarwin`, to be consistent with the other conditionals in that file